src/bp/nlu-server/validation/schemas.ts

```diff
@@ -0,0 +1,105 @@
+import Joi from 'joi'
+
+import { DEFAULT_DUCK_SERVER, DEFAULT_LANG_SERVER, DEFAULT_LANG_SOURCES } from '../config'
+
+const EnumOccurenceSchema = Joi.object({
+  name: Joi.string().required(), // ex: 'Paris', 'Montreal', 'Québec'
+  synonyms: Joi.array() // ex: 'La Ville des lumières', 'City of Paris'
+    .items(Joi.string())
+    .optional()
+    .default([])
+})
+
+const EnumSchema = Joi.object().keys({
+  name: Joi.string().required(), // ex: 'cities'
+  values: Joi.array()
+    .items(EnumOccurenceSchema)
+    .required()
+    .min(1),
+  fuzzy: Joi.number().default(0.9)
+})
+
+// TODO: maybe add some field to tell if information is sensitive (secret)...
```

Comment:
That's an applicative problem, I think we don't this should be done here

Reply:
I don't understand what you mean? Should we do it or not?